src/OOVPADatabase/D3D8LTCG/3925.inl

```diff
@@ -0,0 +1,44 @@
+// ******************************************************************
+// *
+// *   OOVPADatabase->D3D8LTCG->3911.inl
```

Comment:
3911.inl comment does not align with 3925.inl file. However, relative file path is going to be remove. So, let's remove this line as well.